experimental/PropertyDDS/packages/property-properties/src/properties/abstractStaticCollectionProperty.js

```diff
@@ -35,7 +35,7 @@ export class AbstractStaticCollectionProperty extends BaseProperty {
 	/**
 	 * Returns the sub-property having the given name, or following the given paths, in this property.
 	 *
-	 * @param {string|number|array<string|number>} in_ids - The ID or IDs of the property or an array of IDs if an array
+	 * @param {string | number | Array<string | number>} in_ids - The ID or IDs of the property or an array of IDs if an array
```

Comment:
When TypeScript cannot resolve the types declared in JSDoc comments it interprets these as 'unknown', some of which are starting to be caught by TypeScript 5's improved type checks.

(Here the problem is that 'Array' needs to be capitalized.)